Clarify Use of asterisk for custom operator definitions (#11814)
dotnet/docs: docs/fsharp/style-guide/formatting.md

let (!>) x f = f x
```

> For any custom operator with more than one character that starts with `*`, you'll need to add a white space to the beginning of the definition to avoid a compiler ambiguity. Because of this, it's recommended that you simply surround the definitions of all operators with a single white-space character.

Reviewer: A suggestion: 

```suggestion
For any custom operator that starts with `*` and that has more than one character, you need to add a white space to the beginning of the definition to avoid a compiler ambiguity. Because of this, we recommend that you simply surround the definitions of all operators with a single white-space character.
```
